builds/flu/flu_titers.py

```diff
@@ -189,7 +206,7 @@ def glycosylation_count(aa):
     # }
 
 
-def H3N2_scores(runner, tree, epitope_mask_file, epitope_mask_version='wolf'):
+def IAV_scores(runner, tree, mask_file, epitope_mask_version='wolf'):
```

Comment:
Also feels strange that  `avg_age` and `age_score` are calculated under `IAV_scores`. I had been thinking of "scores" as a function of a sequence (alongside mask, or otherwise). Age derives from a different bit of metadata. I'd move this to `flu.process`. Or perhaps to its own class.